spec/copy_spec.lua

```diff
@@ -51,24 +49,24 @@ describe("Test the copy functionality #copy", function()
         assert.are.equals("target/Faketorio-test-mod_0.1.0", faketorio.output_folder)
 
         for _, file in pairs(busted.collect_file_names("src")) do
-            file = string.gsub(file, "src", "factorio/Faketorio-test-mod_0.1.0")
+            file = string.gsub(file, "src", "target/Faketorio-test-mod_0.1.0")
```

Comment:
This changes what is tested drastically.
The target folder is just the temporary working directory where the mod is assembled and appended with tests.

The `factorio`folder serves as a fake mod folder to check that the copy command actually does the right thing, which is copy all the files of the mod to the mod directory of factorio.

Reply:
I broke this by accident when I changed the implicit location of the config file, and then thought the test was wrong. I've reverted those changes and made the test config file path explicit.